eZ/Publish/Core/Persistence/Cache/Tags/TagIdentifierPatterns.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\Core\Persistence\Cache\Tags;
+
+final class TagIdentifierPatterns
```

Comment:
`@internal`
but as mentioned in prev comment, I don't like the solution here. Any addition of a tag will require modifying the code here which does not scale very well and violates open closed principle.

Reply:
Resolving this, as the approach will be reworked.

Reply:
This class is not needed anymore.

Reply:
@konradoboza Is it still used somewhere? How does it relate to `ibexa.persistence.cache.tag_patterns` configuration?

Reply:
It is not needed, I wrongly understood @adamwojs comment. Will remove it.